core/storage/app_feedback_report/gae_models.py

```diff
@@ -245,6 +257,7 @@ def _generate_id(cls, platform, submitted_on_datetime):
 
     @staticmethod
     def get_deletion_policy():
+        # type: () -> Any
```

Comment:
Try to avoid Any.

Reply:
This is because of Enums. Explained in PR description and other comment. Thanks!

Reply:
See above.

Reply:
Added in Doc